cmake/modules/CheckCompiler.cmake

```diff
@@ -198,7 +198,7 @@ if(CMAKE_SYSTEM_NAME MATCHES Linux)
 elseif(APPLE)
   include(SetUpMacOS)
 elseif(WIN32)
-  include(SetupWindows)
```

Comment:
0_0 wow!

Reply:
cmake is mind-boggling. variable names are case-sensitive but commands and some other things are not.